string color1
string color2
string color3
---
bool valid